src/Features/Core/Diagnostics/UpdateArgsId.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Diagnostics
+{
+    internal class UpdateArgsId
+    {
+        public readonly DiagnosticAnalyzer Analyzer;
+
+        /// <summary>
+        /// Base type for <see cref="DiagnosticsUpdatedArgs.Id"/> for live diagnostic
+        /// </summary>
+        /// <param name="analyzer"></param>
```

Comment:
Consider filling in the parameter description or removing the empty XML element.